src/main/java/HSAnimal/demo/controller/SurveyController.java

```diff
@@ -0,0 +1,39 @@
+package HSAnimal.demo.controller;
+
+import HSAnimal.demo.DTO.UserKeywordsDTO;
+import HSAnimal.demo.domain.Questions;
+import HSAnimal.demo.service.SurveyService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class SurveyController {
+    private final SurveyService surveyService;
+
+    public SurveyController(SurveyService surveyService){
+        this.surveyService = surveyService;
+    }
+
+    @GetMapping("/{user_id}/survey")
+    public List<String> questionsList() {
+        List<Questions> questionsList = surveyService.getAllQuestions();
+        List<String> contentList = new ArrayList<>();
+
+        for (Questions question : questionsList) {
+            String content = question.getContent();
+            contentList.add(content);
+        }
+        return contentList;
+    }
+
+    @PostMapping("/{user_id}/survey")
+    public String getOptionsList(@RequestBody List<UserKeywordsDTO> userKeywordsList,@PathVariable String user_id) {
+        for (UserKeywordsDTO userKeywordsDTO : userKeywordsList) {
+            surveyService.keywordsRegister(userKeywordsDTO, user_id);
+        }
+        return "/{user_id}/match 링크로 이동 하고싶은데..";
+    }
```

Comment:
PostMapping의 경우 create (201) status code 와 함께 Location 헤더에 redirect url을 반환해줄 수 있습니다.
클라이언트는 해당 Location 헤더의 값을 읽어 해당 경로로 리다이렉트 하도록 구성할 수 있습니다.

- [참고 문서](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location)

따라서 백엔드에서는 다음과 같이 구성할 수 있습니다.

```suggestion
    @PostMapping("/{user_id}/survey")
    public ResponseEntity<Void> getOptionsList(@RequestBody List<UserKeywordsDTO> userKeywordsList,@PathVariable String user_id) {
        for (UserKeywordsDTO userKeywordsDTO : userKeywordsList) {
            surveyService.keywordsRegister(userKeywordsDTO, user_id);
        }
        return ResponseEntity.created(URI.create(String.format("/%d/match", user_id)).build()
    }
```

Reply:
몰라서 찾아보고 있었는데 감사합니다!